cluster/gce/util.sh

```diff
@@ -241,7 +241,7 @@ function copy-to-staging() {
 
   echo "${hash}" > "${tar}.sha512"
   gsutil -m -q -h "Cache-Control:private, max-age=0" cp "${tar}" "${tar}.sha512" "${staging_path}"
-  gsutil -m acl ch -g all:R "${gs_url}" "${gs_url}.sha512" >/dev/null 2>&1
+  gsutil -m acl ch -g all:R "${gs_url}" "${gs_url}.sha512" >/dev/null 2>&1 || true
```

Comment:
Is this line specific to windows? I wouldn't have expected to ignore an error here

Reply:
Admittedly this isn't the best way to handle this issue. This is the fix in v1.19+. The proper solution is to set public-read on the bucket if this operation fails but that also has bad side effects. All in all there's not really a great workaround here other than just making this optional. Lastly, this probably does impact Linux as well. I was mainly testing Windows clusters but this silent failure happens before cluster bringup.

Reply:
Ping this still breaks. To reiterate this is the logic that newer versions of Kubernetes does so this is a backport of the awful fix.

Reply:
this is the PR where it was introduced: https://github.com/kubernetes/kubernetes/pull/92668

Reply:
@jeremyje do you need to grab a second line from the source PR?